templates/dialog/automatic-fire.html

```diff
@@ -1,8 +1,13 @@
-<div style="height: 30px">
-    <label>{{localize "YZECORIOLIS.AutomaticFire"}}</label>
+<div>
+    <label style="position:relative; bottom:6px;">{{localize "YZECORIOLIS.AutomaticFire"}}</label>
     <input type='checkbox' name='automaticFire' data-dtype='Boolean' />
-    <span class='ignoredOnes' hidden style="display: none">
-    <label>{{localize "YZECORIOLIS.AutomaticFireIgnoredOnes"}} :</label>
-    <input name='numberOfIgnoredOnes' style="width: 20px" type="number" min="0" max="2" data-dtype='Number' value="0"/>
+    <span class='ignoredOnes' hidden style="display: none; position:relative; bottom:6px;"> ({{localize "YZECORIOLIS.AutomaticFireModifier"}})
+      <br/><br/>
+      <label style="position:relative; bottom:6px;">{{localize "YZECORIOLIS.Talent"}}: {{localize "YZECORIOLIS.TalentMachinegunner"}}?</label>
+      <input type='checkbox' name='maschineGunner' data-dtype='Boolean' />
+      <br/>
+      <label style="position:relative; bottom:6px;">{{localize "YZECORIOLIS.Weapon"}} {{localize "YZECORIOLIS.Feature"}}: {{localize "YZECORIOLIS.FeatureWeaponHighCapacity"}}?</label>
```

Comment:
Unfortunately, I don't think this will work for all languages.

Reply:
maybe just `{{localize "YZECORIOLIS.Weapon"}}: {{localize "YZECORIOLIS.FeatureWeaponHighCapacity"}}`

Reply:
Hmm thats why my first idea was 
`{{localize "YZECORIOLIS.Feature"}} ({{localize "YZECORIOLIS.Weapon"}}):`
since that should be specific but not language-breaking 🤔 

But yours is great as well.